src/08-accessing.js

```diff
@@ -148,7 +166,10 @@ function getCityAndState(person) {}
  *  getFullAddress(person);
  *  //> "697 Pine Drive 2A, Staten Island, NY, 10306"
  */
-function getFullAddress(person) {}
+function getFullAddress(person) {
+  let fullAddy = `${person.address.street} ${person.address.unit}, ${person.address.city}, ${person.address.state}, ${person.address.zip}`;
```

Comment:
For long chained template strings we should move to a new line.  Remember tempalte literals can be spread across multiple lines.